Source/Core/VideoBackends/D3D/D3DUtil.cpp

```diff
@@ -97,8 +97,40 @@ class UtilVertexBuffer
   std::list<bool*> observers;
 };
 
-CD3DFont font;
-UtilVertexBuffer* util_vbuf = nullptr;
+// Font creation flags
+#define D3DFONT_BOLD 0x0001
+#define D3DFONT_ITALIC 0x0002
+
+// Font rendering flags
+#define D3DFONT_CENTERED 0x0001
```

Comment:
Are these three defines actually used for anything?

Reply:
Not as far as I can see.

Reply:
Removed them in a separate commit